lib/electric_slide/agent.rb

```diff
@@ -2,7 +2,7 @@
 
 class ElectricSlide
   class Agent
-    attr_accessor :id, :address, :presence, :call, :queue
+    attr_accessor :id, :address, :presence, :call, :queue, :removed_by
```

Comment:
I'd rather not store this on the agent, but rather pass it via a presence update only.